daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/RuntimeData.scala

```diff
@@ -618,8 +622,9 @@ final class ElementRuntimeData(
   @TransientParam targetNamespaceArg: => NS,
   @TransientParam thisElementsNamespaceArg: => NS,
   @TransientParam optSimpleTypeRuntimeDataArg: => Option[SimpleTypeRuntimeData],
-  @TransientParam minOccursArg: => Option[Int],
-  @TransientParam maxOccursArg: => Option[Int],
+  @TransientParam minOccursArg: => Long,
+  @TransientParam maxOccursArg: => Long,
```

Comment:
Can we actually support more more than IntMax array elements. I believe we still use an Array as the underlying data structure for storing arrays so I think these values are still limited to Int's.